torch/utils/data/dataset.py

```diff
@@ -194,16 +194,21 @@ def __init__(self, datasets):
     def __len__(self):
         return self.cumulative_sizes[-1]
 
-    def __getitem__(self, idx):
-        if idx < 0:
-            if -idx > len(self):
-                raise ValueError("absolute value of index should not exceed dataset length")
-            idx = len(self) + idx
+    def get_idxs(self, idx):
```

Comment:
We should add a docstring to this so that users know what it does.

Reply:
Added a docstring in 718999e